app/src/main/java/ca/llamabagel/transpo/TranspoApplication.kt

```diff
@@ -32,5 +31,11 @@ class TranspoApplication : Application(), DaggerComponentProvider {
         component.inject(this)
         // val workerFactory = DaggerApplicationComponent.create()
         WorkManager.initialize(this, Configuration.Builder().setWorkerFactory(workerFactory).build())
+
+        if (android.os.Build.VERSION.SDK_INT >= ANDROID_O_SDK) {
```

Comment:
There will be a proper theme/dark mode setting available in the app settings which will be implemented later. Users will be able to override light or dark themes or choose to follow system.

In other words, this shouldn't be handled in the Application class.

Reply:
yes this is temporary until we implement that